pylasu/model/model.py

```diff
@@ -95,13 +96,56 @@ def provides_nodes(decl_type):
         return isinstance(decl_type, type) and issubclass(decl_type, Node)
 
 
+def get_only_type_arg(decl_type):
+    type_args = get_type_arguments(decl_type)
+    if len(type_args) == 1:
```

Comment:
If the type_args are more than one should we throw an exception perhaps?
If not, I would add a short document to explain it